### DISCUSSION TITLE
What's the state on i18n ?

### DISCUSSION THREAD

**Discussion by Baptistemontan** [leptos-rs/leptos#1495] [General]
I'm currently working on a website and being french I would like to have both french and english available, I did my own very basic thing here  https://github.com/Baptistemontan/leptos_i18n for my website (https://github.com/Baptistemontan/hex-chess-app) but I'm no expert in browser specific thing that translation needs, is there a crate being made for internalization with leptos ? I can't find any.

**Comment by CedricGARVENES** [comment]
Hi @Baptistemontan ,
I'm also French and I'm just starting out with leptos. I'm trying to make a fullstack website with leptos(Frontend) and Axum (Backend). How does leptos_i18n work? And what is i18n?

**Reply by Baptistemontan** [reply]
Hello!  "i18n" is an abreviation for "internationalisation", this is the process to support multiple languages in an application.
`leptos_i18n` is a library (that I maintain) that tries to help as much as possible the process of internalizing a leptos application, you can found more information on the readme or the documention (such as the [book](https://baptistemontan.github.io/leptos_i18n/))
